content/enterprise/system-overview/capacity.mdx

```diff
@@ -25,12 +25,12 @@ After factoring in the memory needed to run the base services that make up the a
 The settings for per-run memory and concurrency are available in the dashboard on port 8800, on the Settings page, under the Capacity section. They can also be set via
 the [application settings JSON file when using the automated install procedure](/enterprise/install/automated/automating-the-installer#available-settings).
 
-## Increasing Capacity
+## Increasing Concurrency
 
-To increase the number of concurrent runs, adjust the **Capacity** setting. Note that this setting is not limited by
+To increase the number of concurrent runs, adjust the `capacity_concurrency` setting. Note that this setting is not limited by
```

Comment:
This is a nitpick, but I think we shouldn't start with "Note that" unless we're going to put something in a Note block. Otherwise, we can just make it a statement. 

```suggestion
To increase the number of concurrent runs, adjust the `capacity_concurrency` setting. This setting is not limited by
```